src/CsvTextEditor/ViewModels/SettingsViewModel.cs

```diff
@@ -56,6 +60,21 @@ private void OnOpenApplicationDataDirectoryExecute()
             _manageUserDataService.OpenApplicationDataDirectory();
         }
 
+        public TaskCommand PickEditor { get; private set; }
+
+        private async Task PickEditorExecuteAsync()
+        {
+            var resolver = ServiceLocator.Default.GetDependencyResolver();
+            var openFileService = resolver.Resolve<IOpenFileService>();
+
+            openFileService.Filter = "Program Files (*.exe)|*exe";
+            openFileService.IsMultiSelect = false;
+
+            var result = await openFileService.DetermineFileAsync();
```

Comment:
I think we should do something like this instead:

if (await _openFileService.DeterminFileAsync() ?? false)

(because the user can cancel)